### PR TITLE
CVSL-1786 Returning Approval view licences

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/LicenceSummaryApproverView.kt [hunk 1]
```diff
@@ -8,8 +8,8 @@ import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
 import java.time.LocalDate
 import java.time.LocalDateTime
 
-@Schema(description = "Response object which summarises an approved licence")
-data class ApprovedLicenceSummary(
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/LicenceSummaryApproverView.kt:12]: Renamed this as its used for licences that have not been approved yet

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt [hunk 2]
```diff
@@ -309,7 +309,7 @@ class LicenceController(
   fun getRecentlyApprovedLicences(
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt:309]: now returning for getRecentlyApprovedLicences - this technically isn't required but is conceptually simpler: using the same bespoke representation for the approver in both tabs.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt [hunk 3]
```diff
@@ -329,7 +342,11 @@ class LicenceServiceTest {
       actualReleaseDate = LocalDate.now().minusDays(1),
       conditionalReleaseDate = LocalDate.now(),
       approvedByName = "jim smith",
+      statusCode = LicenceStatus.INACTIVE,
```
Comment (andrewrlee) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt:345]: tried to make the data a little bit more realistic

**Review by nishanthmahasamudram — APPROVED** [review]
LGTM

**Pull request merged**